soknad/src/test/java/no/nav/k9/søknad/ytelse/omsorgspenger/v1/OmsorgspengerUtbetalingValidatorTest.java

```diff
@@ -303,6 +303,26 @@ void korrigering_av_perioder_innenfor_eksisterende_perioder_skal_ikke_feile() {
         assertThat(feil).hasSize(0);
     }
 
+    @Test
+    void korrigering_av_perioder_innenfor_eksisterende_perioder_som_strekker_over_helg_skal_ikke_feile() {
```

Comment:
Dette har vel egentlig ikke noe med helg å gjøre, er vel generelt at hvis man skal korrigere to perioder, og det finnes minst en dag med opphold mellom, så feilet det?

Reply:
Det stemmer. Har endret navn på testen.

Poenget var å formulere at periodene var oppdelt pga. helg :)